Formula/wireshark.rb

```diff
@@ -18,29 +18,32 @@ class Wireshark < Formula
   option "with-gtk+", "Build the wireshark command with gtk+"
   option "with-qt", "Build the wireshark command with Qt (can be used with or without either GTK option)"
   option "with-headers", "Install Wireshark library headers for plug-in development"
+  option "with-nghttp2", "Enable HTTP/2 header dissection"
 
+  depends_on "pkg-config" => :build
```

Comment:
This is not required, CMake can find the required libraries on macOS without pkg-config (it is only required for autotools)

Reply:
done.